evap/student/templates/student_global_reward.html

```diff
@@ -0,0 +1,53 @@
+{% if global_rewards %}
+    <div class="card mb-3">
+        <div class="card-header d-flex">
+            {% translate "Fundraising" %}
+            {% if global_rewards.last_vote_datetime %}
+                <div class="ms-auto fw-normal fst-italic">
+                    {% blocktranslate trimmed with time_interval=global_rewards.last_vote_datetime|timesince %}
+                        Last evaluation: {{ time_interval }} ago
+                    {% endblocktranslate %}
+                </div>
+            {% endif %}
+        </div>
+        <div class="card-body">
+            <div class="global-rewards-progress-bar pt-2 px-4">
+                <span class="progress-container">
+                    <span class="progress m-0 bg-info">
+                        <span class="progress-bar bg-primary width-percent-{% widthratio global_rewards.bar_width_votes global_rewards.max_reward_votes 100 %}"></span>
+                        <span class="progress-bartext">
+                            {% widthratio global_rewards.vote_count global_rewards.participation_count 100 as percent %}
+                            {% blocktranslate trimmed count votes=global_rewards.vote_count %}
+                                {{ votes }} submitted evaluation ({{ percent }}%)
+                            {% plural %}
+                                {{ votes }} submitted evaluations ({{ percent }}%)
+                            {% endblocktranslate %}
+                        </span>
+                    </span>
+                </span>
+                {% for reward in global_rewards.rewards_with_progress %}
+                    <div class="position-relative progress-step left-percent-{% widthratio reward.progress 1 100 %}">
+                        <div class="mx-auto seperator"></div>
+                        <div class="small text-center">
+                            {% widthratio reward.vote_ratio 1 100 as percent %}
+                            {% blocktranslate %}{{ percent }}%{% endblocktranslate %}
```

Comment:
This shouldn't need to be translated, right?
```suggestion
                            {% widthratio reward.vote_ratio 1 100 %}%
```